plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/utils/SystemVariablesResolver.java

```diff
@@ -63,6 +74,12 @@ public String get(String name) {
                     return "127.0.0.1";
                 }
             default:
+                if (configurationFile != null) {
+                    final Object o = configurationFile.get(name);
+                    if (o != null) {
+                        return o.toString();
+                    }
+                }
```

Comment:
I believe the variables from this property file have higher priority than the other variables.